ci/builders/README.md

```diff
@@ -154,231 +156,252 @@ An archive component is used to tell the recipes which artifacts are generated b
 
 Description of the fields:
 
-* **Name:** Used to identify the archive inside CAS.
-* **Base\_path:** The portion of the path to remove before uploading to its destination. In the example the base\_path **“out/host\_debug/zip\_archives”** will be extracted from the include path **"out/host\_debug/zip\_archives/linux-x64/artifacts.zip"** before uploading to GCS, e.g. &lt;bucket>/flutter/&lt;commit>/linux-x64/artifacts.zip.
-* **Type:** The type of storage to use. Currently only **“GCS”** and **“CAS”** are supported.
-* **Include\_paths:** A list of strings representing paths to artifacts generated by the build that need to be uploaded to a given destination.
-
-##### Drone\_dimensions
-
-A list of strings with key value pairs separated by an equal sign. These dimensions are used to select the bot where the sub-build will be running.
-
-```
-"drone_dimensions": [          "device_type=none",
+* **name:** - by default the entire build output is uploaded to CAS.
+ `name` is used to associate the CAS hash to a value that can be referenced
+ later as a dependency of global tests. Name is also used to select the folder
+ from within src/out to upload to CAS. e.g if the build generates
+ src/out/host_debug name must be `host_debug`.
+* **base\_path:** - the portion of the path to remove from the full path before
+ uploading to its final destination. In the example the above the
+ base\_path **“out/host\_debug/zip\_archives”** will be removed from the
+ include path **"out/host\_debug/zip\_archives/linux-x64/artifacts.zip"**
+ before uploading to GCS, e.g.
+ &lt;bucket&gt;/flutter/&lt;commit>/linux-x64/artifacts.zip.
+* **Type:** - the type of storage to use. Currently only **“GCS”** and
+**“CAS”** are supported. Supported values for this property are "gcs" for GCS
+and "cas" to archive to CAS.
+* **Include\_paths:** - a list of strings with the paths to be uploaded to a
+given destination.
+* **cas_archive** - a boolean value indicating whether the build output will
+be archived to CAS or not. The default value is false.
```

Comment:
The default value is true?

Reply:
Thanks, yes the correct default is true.

Reply:
Why do we have both `type` which can be `CAS`, and a `cas_archive` boolean field? Seems confusing.

What happens if `type` is specified as `CAS`, but `cas_archive` is `false`?